### ISSUE TITLE
Cash game structure

### ISSUE THREAD

**Issue by CharlieHess** [CharlieHess/slack-poker-bot#9]
Right now the game is structured in tournament form; each player starts with $200 and blinds are fixed at $1/2. It would be cool to have the option for a cash game, where players can leave at any time or re-buy.

**Comment by CaptainPalapa** [comment]
Agreed, I couldn't quite figure out how, once you do Yes" to join the game...not sure how to get out of the game?

**Comment by richardtape** [comment]
Also might be nice, in tournament style, to up the blinds every x-minutes. Probably have to make it configurable.
